KNOWN_ISSUES.md

```diff
@@ -127,7 +128,51 @@ kubectl get --raw /api/v1/nodes/$NODE_NAME/proxy/metrics/resource
   **Workaround**
   
   Upgrade Docker to v19.03
-  
+
+## HPA is unable to get resource utilization
+
+**Symptoms**
+
+
+When running Deployment with horizontal autoscaling, HPA fails to compute replica count and reports error `failed to get memory utilization: missing request for memory`, even though Deployment sets resource requests. For example when running:
+
+$ kubectl describe hpa 
```

Comment:
Can you format the commands in ``` (I was not able to do that in suggestions due to Github limitations)

Reply:
Sorry, I didn't notice it just now. has already been updated